tests/e2e/configuration-reload/chainsaw-test.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: configuration-reload
+spec:
+  description: |
+    This e2e test verifies collector configuration reload functionality.
+    It creates an action and verifies that the collector configuration is updated.
+  skipDelete: true
+  steps:
+    - name: Prepare destination
+      try:
+        - apply:
+            file: ../../common/apply/simple-trace-db-deployment.yaml
+
+    - name: Install Odigos
+      try:
+        - script:
+            content: |
+              ../../../cli/odigos install --namespace odigos-test --version e2e-test
+              ../../common/verify_odigos_installation.sh odigos-test
+            timeout: 4m
+        - assert:
+            timeout: 3m
+            file: ../../common/assert/odigos-installed.yaml
+
+    - name: Assert Trace DB is up
+      try:
+        - assert:
+            timeout: 1m
+            file: ../../common/assert/simple-trace-db-running.yaml
+
+    - name: Add Destination
+      try:
+        - apply:
+            file: ../../common/apply/add-simple-trace-db-destination.yaml
+
+    - name: Wait for destination to be ready
+      try:
+        - script:
+            timeout: 60s
+            content: |
+              echo "Waiting for destination to be processed..."
+              sleep 10
+              echo "Destination ready"
+
+    - name: Create cluster info action
+      try:
+        - apply:
+            file: 03-create-action.yaml
```

Comment:
the `03` is meant to follow `01`, `02` and help to identify the stage of the test.